.vscode/launch.json

```diff
@@ -6,34 +6,35 @@
       "type": "coreclr",
       "request": "launch",
       "preLaunchTask": "build (CopilotChatWebApi)",
-      // If you have changed target frameworks, make sure to update the program path.
```

Comment:
Seem unrelated. Revert these comment removals?

Reply:
i think the intellisense did it, added comments back